src/tests/Sort.test.js

```diff
@@ -0,0 +1,46 @@
+const { app } = require('../pages/Application'); 
+
+
+describe('Inventory Sorting', () => {
+    before(async () => {
+        await app.login.navigate();
+        await app.login.performLogin('standard_user', 'secret_sauce');
+    });
+
+    it('should sort items by name (Z to A)', async () => {
+        await app.inventory.navigate();
+
+        const inventoryItems = await app.inventory.inventoryItems;
+        if (!inventoryItems.length) {
+            throw new Error('No inventory items found on the page');
+        }
+
+        const initialNames = [];
+        for (const item of inventoryItems) {
+            const itemName = await item.$('.inventory_item_name').getText();
```

Comment:
It is generally bad practice to define locators in the test cos they are subject to change

Reply:
I have added to the Inventory.js page a method 🤞 hope I have understood correctly.
  getItemName(item) {
        return item.$('.inventory_item_name');
    }